CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Bugfixes
 
 * Update Go runtime to 1.20.3. [#987](https://github.com/elastic/package-registry/pull/987)
+* Update Go runtime to 1.20.4. [#1002](https://github.com/elastic/package-registry/pull/1002)
```

Comment:
I was thinking to just keep one changelog entry for this, like:
```
* Update Go runtime to 1.20.4. [#1002](https://github.com/elastic/package-registry/pull/1002)
```

or maybe with the two PR links
```
* Update Go runtime to 1.20.4. [#987](https://github.com/elastic/package-registry/pull/987) [#1002](https://github.com/elastic/package-registry/pull/1002)
```

WDYT ?

Reply:
Yes, keep a single entry for this. Having both links sounds good.